README.md

```diff
@@ -50,6 +50,27 @@ subscribers to those tags via the external services.
 * Check that the configuration in `config/database.yml` is correct
 * Run `bundle exec rake db:setup` to load the database
 
+#### Troubleshooting
+
+If you get this error when setting up the database:
+
+```
+ActiveRecord::StatementInvalid: PG::InsufficientPrivilege: ERROR:  permission denied to create extension "uuid-ossp"
+HINT:  Must be superuser to create this extension.
+: CREATE EXTENSION IF NOT EXISTS "uuid-ossp"
+/var/govuk/email-alert-api/db/schema.rb:17:in `block in <main>'
+/var/govuk/email-alert-api/db/schema.rb:13:in `<main>'
+/usr/lib/rbenv/versions/2.6.1/bin/bundle:23:in `load'
+/usr/lib/rbenv/versions/2.6.1/bin/bundle:23:in `<main>'
+```
+
+Run the following to be able to run psql as a superuser:
+
+```
+sudo -u postgres psql postgres
+ALTER USER vagrant WITH SUPERUSER;
```

Comment:
I thought govuk-puppet was meant to do something about this [1]... maybe running `govuk_puppet` would work? I wouldn't be surprised if govuk-puppet would revert that role change.

1: https://github.com/alphagov/govuk-puppet/blob/4c27016f72e5b2a661393c0cb148d11b94599d95/modules/govuk/manifests/apps/email_alert_api/db.pp#L38

Reply:
Hmmm. I didn't try that. I'll update the docs.